lib/content-types/load.js

```diff
@@ -39,6 +39,17 @@ const check = (type) => {
   if (type.id !== slugify(type.id)) {
     return `${type.id} needs to be written in kebab case (e.g. ${slugify(type.id)})`;
   }
+
+  if (type.hasOwnProperty('identifier')) {
+    if (typeof type.identifier !== 'boolean') {
+      return 'Content type identifier must be a boolean';
```

Comment:
You are reading an old commit. Please see the latest here:
https://github.com/punchcard-cms/content-types/pull/72/files

This entire `check` function is gone in the push from yesterday